docs/releases/3.0.txt

```diff
@@ -315,6 +315,12 @@ backends.
   ``can_return_ids_from_bulk_insert`` are renamed to
   ``can_return_columns_from_insert`` and ``can_return_rows_from_bulk_insert``.
 
+* Database functions now handle :class:`datetime.timezone` formats when created
+  using :class:`datetime.timedelta` instances (e.g.
+  ``timezone(timedelta(hours=5))``. Third-party backends should handle this
```

Comment:
I think this is missing a closing `)`

Plus **maybe** adding the string that `timezone(timedelta(hours=5))` results in (which is the mentioned _format_ that needs to be handled)